docs/source/Lines_functionality.ipynb

```diff
@@ -0,0 +1,1861 @@
+{
```

Comment:
<p>It is interesting that the smoothed one also has a lot of points but that is more about our data than the package itself.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OSOceanAcoustics/echoregions/pull/184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>